modules/compiler/targets/host/source/target.cpp

```diff
@@ -180,6 +231,20 @@ compiler::Result HostTarget::initWithBuiltins(
     }
   } else if (triple.isX86()) {
     CPUName = "x86-64-v3";  // Default only, may be overridden below.
+  } else if (triple.isRISCV()) {
+    if (triple.isArch32Bit()) {
+      Abi = "ilp32d";
+      CPUName = "generic-rv32";
+    } else {
+      Abi = "lp64d";
+      CPUName = "generic-rv64";
+    }
+    // We assume the following features must be supported.
```

Comment:
Why these? What is this based on?

Reply:
I can't recall the origins but it's what we use for refsi.

Reply:
I think they're a fair default but could maybe use documentation to explain?

* We can't do anything without atomics (`a`).
* We *could* do stuff without multiplication/division (`m`) but we'd need extra compiler-rt support to do the most basic things and it'd be really slow.
* Assuming hardware support for fp32/fp64 (`f`/`d`) seems sensible to me. Same problem as above with compiler-rt, software emulation and performance.

All of these should probably be overrideable, as the TODO suggests, but I don't think it's required for now.